### ISSUE TITLE
As Robot, using Fixed Payout after using 1 Jackpot ability does not bring up the Jackpot UI again

### ISSUE THREAD

**Issue by ShiftedClock** [TerryCavanagh/diceydungeons.com#422]
How to recreate: As Robot, get a Jackpot and use a Jackpot ability (so the Jackpot UI goes away). Then use the Fixed Payout item (which says "Use an additional Jackpot ability"). The Jackpot UI does not come back up again.

This defies expectations slightly, because when the Jackpot UI is up (and you haven't used a Jackpot ability) and you use Fixed Payout, the Jackpot ability changes to say "Jackpot +1". So there's a precedent that Fixed Payout can be used after getting the Jackpot to get the extra ability.

Additionally, there are some turns where it makes sense to use a Jackpot ability before using Fixed Payout. For instance, if you use the One More Roll Jackpot ability and get an even number, it can be put into Fixed Payout.

This may seem like a "cheap tactic" at first, but it's really a risk/reward tradeoff, because you may not get an even die from One More Roll. This in turn can be mitigated by saving a bump/nudge ability for after Jackpot, since you'll always be able to manipulate the extra roll into an even number. Which is a neat interaction.

**Comment by TerryCavanagh** [comment]
I agree! Want to make this change soon, thank you for creating an issue :)
